lib/application_configuration.rb

```diff
@@ -13,7 +13,7 @@ def configured_value(config_path, options = {})
     # a query will yield false (as a configured value), so we cannot
     # use standard .present? or .blank?. We also cannot use .empty?
     # because TrueClass and FalseClass do not support it
-    if value != {} then value
+    if !value.nil? then value
```

Comment:
It definitely did not whenever I was writing this I'm about 95% sure of this. I remember something rendering on the page with the value `{}` because it passed this check.